Resources/Prototypes/_LostParadise/Entities/Objects/Consumable/Drinks/drinks.yml

```
@@ -0,0 +1,31 @@
+  suffix: cargo gargle blaster
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        maxVol: 30
+        reagents:
+        - ReagentId: LPPCargoGargleBlaster
+          Quantity: 30
+  - type: Icon
+    sprite: Objects/Consumable/Drinks/gargleblasterglass.rsi
+    state: icon
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)